src/discrete-layer/components/layer-details/utils.ts

```diff
@@ -80,8 +80,10 @@ export const getFlatEntityDescriptors = (
   return flat;
 };
 
-export const getBasicType = (fieldName: FieldInfoName, typename: string): string => {
+export const getBasicType = (fieldName: FieldInfoName, typename: string, lookupTableKey: string | null = null): string => {
```

Comment:
change to:
lookupTableKey?: string
 so you wont need the default assertion to null.
 then check for existence.